client/consul/sync.go

```diff
@@ -353,7 +398,28 @@ func (c *ConsulService) filterConsulChecks(chks map[string]*consul.AgentCheck) m
 	return nomadChecks
 }
 
+// consulPresent indicates whether the consul agent is responding
 func (c *ConsulService) consulPresent() bool {
 	_, err := c.client.Agent().Self()
 	return err == nil
 }
+
+// runCheck runs a check and updates the corresponding ttl check in consul
+func (c *ConsulService) runCheck(check Check) {
+	res := check.Run()
+	state := consul.HealthCritical
+	output := res.Output
+	if res.Err != nil {
+		output = res.Err.Error()
+	}
+	if res.ExitCode == 0 {
```

Comment:
An option that I think is cleaner
```
var state string
switch res.ExitCode {
case 0:
  state = consul.HealthPassing
case 1:
  state = consul.HealthWarning
default:
   state = consul.HealthCritical
}
```

Otherwise change the two ifs into an if/else